src/components/date-picker/date-picker.js

```diff
@@ -0,0 +1,227 @@
+    });
+    fromDate.addEventListener('focus', () => {
+      fromDate.value = fromDate.value.replace(/\s+/g, '');
+    });
+    toDate.addEventListener('focus', () => {
+      toDate.value = toDate.value.replace(/\s+/g, '');
+    });
+  }
+
+  _updateDatePickerPosition = (calendar) => {
+    const cal = calendar;
+    cal.calendarContainer.style.top = '67px';
+    cal.calendarContainer.style.left = '0px';
+  }
+
+  _updateClassNames = (calendar) => {
+    const calendarContainer = calendar.calendarContainer;
+    calendarContainer.classList.add(this.options.classCalendarContainer);
+    calendarContainer.querySelector('.flatpickr-month').classList.add(this.options.classMonth);
+    calendarContainer.querySelector('.flatpickr-weekdays').classList.add(this.options.classWeekdays);
+    calendarContainer.querySelector('.flatpickr-days').classList.add(this.options.classDays);
+    [...calendarContainer.querySelectorAll('.flatpickr-weekday')].forEach((item) => {
+      const currentItem = item;
+      currentItem.innerHTML = currentItem.innerHTML.replace(/\s+/g, '');
+      currentItem.classList.add(this.options.classWeekday);
+      if (currentItem.innerHTML === 'Thu' || currentItem.innerHTML === 'Th') {
+        currentItem.innerHTML = 'Th';
+      } else {
+        currentItem.innerHTML = item.innerHTML.charAt(0);
+      }
+    });
+    [...calendarContainer.querySelectorAll('.flatpickr-day')].forEach((item) => {
+      if ((item.classList.contains('selected') && item.classList.contains('inRange'))
+         || (item.classList.contains('selected') && item.classList.contains('endRange'))) {
+        if (item.previousElementSibling) {
+          item.previousElementSibling.classList.add(this.options.classPrevDay);
+        }
+      }
+      item.classList.add(this.options.classDay);
+    });
+  }
+
+  _updateInputFields = (selectedDates, isRange) => {
+    if (isRange) {
+      if (selectedDates.length === 2) {
+        this.element.querySelector(this.options.selectorDatePickerInputFrom).value = this._formatDate(selectedDates[0]);
+        this.element.querySelector(this.options.selectorDatePickerInputTo).value = this._formatDate(selectedDates[1]);
+      } else {
+        this.element.querySelector(this.options.selectorDatePickerInputFrom).value = this._formatDate(selectedDates[0]);
+      }
+    } else {
+      this.element.querySelector(this.options.selectorDatePickerInput).value = this._formatDate(selectedDates[0]);
+    }
+  }
+
+  _formatDate = (date) => {
+    const month = (date.getMonth() + 1) < 10
+    ? `0${(date.getMonth() + 1)}`
+    : (date.getMonth() + 1);
+    const day = date.getDate() < 10
+    ? `0${date.getDate()}`
+    : date.getDate();
+    return `${month} / ${day} / ${date.getFullYear()}`;
+  }
+
+  _initDatePicker = () => {
+    const date = this.element.querySelector(this.options.selectorDatePickerInput);
+    const calendar = new Flatpickr(date, {
+      allowInput: true,
+      dateFormat: 'm/d/Y',
```

Comment:
It’s a bit of repeating, but, for example, “2017/5/31” is used in Japan instead of “5/31/2017”. Seems that Flatpickr has [custom `formatDate()` config](https://github.com/chmln/flatpickr/commit/a1c3548b7046d07c4d10e0ae8efdb01442337587) as well ass [custom `parseDate()` config](https://github.com/chmln/flatpickr/blob/a1c3548b7046d07c4d10e0ae8efdb01442337587/src/flatpickr.js#L1103), and thus I’d suggest using those - The implementation should use `Into.DateTimeFormat`.